### ISSUE TITLE
Dependency mismatch: libpq-dev needs a different version of libpq5 than pgAdmin4

### ISSUE THREAD

**Issue by gnn** [openego/eGon-data#105]
This discussion was originally started by @ClaraBuettner in https://github.com/openego/eGon-data/issues/103#issuecomment-762693948 and moved here in order to tackle this as a separate issue.

This dependency mismatch is interesting. I'm on Ubuntu 20.04 where this problem doesn't occur. Since I'm not finding `pgAdmin4` for Ubuntu 18.04, I'm guessing, that you didn't get it from the official package sources? I'm just trying to figure out, where the dependency mismatch is coming from. Since modifying the permissions/ownership on stuff under "/var" isn't the most stable approach, instead of creating the "pgadmin" directories under "/var" one can execute the following in a shell, while the virtual environment into which `pgAdmin4` got installed is activated:

```sh
(
PYTHON="python3.9" && \
SUFFIX="../lib/$PYTHON/site-packages/pgadmin4/config_local.py" && \
TARGET="$HOME/.pgadmin" && \
echo "\
SERVER_MODE = False
LOG_FILE = '$TARGET/pgadmin4.log'
SESSION_DB_TARGET = '$TARGET/sessions'
SQLITE_TARGET = '$TARGET/pgadmin4.db'
STORAGE_DIR = '$TARGET/storage'" \
> "$(dirname $(which pgadmin4))/$SUFFIX";
)
```

Change `PYTHON` to match the Python version the virtual environment is using and if you want pgAdmin's files to go somewhere else, change `TARGET` to your liking. Otherwise, you should be able to just copy and paste it.

I also didn't know that `pgAdmin4` was a Python package. Good to know.
Last but not least, is this maybe another argument for installing pgAdmin4 in a Docker container? Or should we just put a writeup of this discussion in the "Troubleshooting" section?

**Comment by ClaraBuettner** [comment]
Yes, I couldn't install pgAdmin4 from the official package sources on Ubuntu 18.04, I followed [these](https://www.pgadmin.org/download/pgadmin-4-apt/) instructions. 
I think it is ok to document how users can install pgadmin4 and not install it in a docker container. 
But if pgadmin4 is a python package, couldn't we add it to the egon-data installation, maybe as an optional package?

**Comment by gnn** [comment]
> Yes, I couldn't install pgAdmin4 from the official package sources on Ubuntu 18.04, I followed [these](https://www.pgadmin.org/download/pgadmin-4-apt/) instructions.
> I think it is ok to document how users can install pgadmin4 and not install it in a docker container.

OK. Then I'll just do this.

> But if pgadmin4 is a python package, couldn't we add it to the egon-data installation, maybe as an optional package?.

In theory, yes. But unfortunately pgAdmin4 pins its `Flask` dependency to `==1.0.2` which is incompatible with Airflow's `flask>=1.1.0`. So both have to live in different virtual environments.
